client/extensions/wp-super-cache/preload-tab.jsx

```diff
@@ -30,28 +30,37 @@ import { isPreloadingCache } from './state/cache/selectors';
  */
 const renderCachePreloadInterval = ( {
 	handleChange,
-	isRequesting,
-	isSaving,
+	isDisabled,
 	preload_interval,
 } ) => (
 	<FormTextInput
 		className="wp-super-cache__preload-interval"
-		disabled={ isRequesting || isSaving }
+		disabled={ isDisabled }
 		min="0"
 		name="preload_interval"
 		onChange={ handleChange( 'preload_interval' ) }
 		step="1"
 		type="number"
-		value={ preload_interval || '' } />
+		value={ preload_interval || '0' } />
```

Comment:
Maybe fallback to `0` (integer), and maybe do it as part of the [variable declaration](https://github.com/Automattic/wp-calypso/pull/14518/files#diff-53f6addaad62845c21e19b56e1c2b43dR34)?